parser/build.gradle.kts

```diff
@@ -21,10 +21,14 @@ tasks.compileJava {
     dependsOn(tasks.generateGrammarSource)
 }
 
+tasks.named("compileKotlin") {
```

Comment:
you can use the DSL here as well; `tasks.compileKotlin`

Reply:
I didn't know if that would work given that it's referencing an automatically-generated task rather than declaring it.